### ISSUE TITLE
[Extension Feature Request] Emoji Search - pop to root after emoji is pasted

### ISSUE THREAD

**Issue by frouo** [raycast/extensions#603]
# Extension – [Emoji Search](https://github.com/raycast/extensions/tree/main/extensions/emoji)

Author: @FezVrasta 

I would prefer that `popToRoot` is called after emoji is pasted to the current window.

Why? Here is my workflow. I have remapped the default macOS emoji shortcut `Ctrl + Cmd + Space`.

* So when I need a emoji, I hit `Ctrl + Cmd + Space`, search and find my emoji, hit `Enter`. Emoji is pasted to the current window. 
* Absolutely perfect. Love it.
* I need another emoji, I hit `Ctrl + Cmd + Space`, search and find my emoji, hit `Enter`. Emoji is pasted to the current window. 
* Truly love it.
* Now I need to open Raycast to find an app or whatever, I hit `Cmd + Space` and
* 💥 I am a bit lost because Raycast opens on Emoji Search 

I would expect that Raycast opens to root (because I hit `Cmd Space`.

**Comment by frouo** [comment]
Discussion [here](https://raycastcommunity.slack.com/archives/C01AC2X0GMN/p1641326659111500)

**Comment by FezVrasta** [comment]
This is what happens to me.

https://user-images.githubusercontent.com/5382443/148210456-9ef8fc89-0472-4cbf-a552-3e4d207c13c2.mp4

**Comment by frouo** [comment]
Yes sorry, indeed it happens if you follow these steps :

1. Raycast hotkey `Cmd + Space` : Raycast opens
2. Cmd hotkey `Ctrl + Cmd + Space` : emoji search opens
3. Hit `Enter` to copy emoji (or `Esc` to close raycast) : Raycast closes
4. Raycast hotkey : emoji search opens (instead of root)

Yes, I should avoid step 1 but sometimes you know, it happens (to me 😅).

**Comment by frouo** [comment]
I am going to close the issue because I am supposed to use my emoji search hotkey.

Sorry I made it a little confusing with the Raycast hotkey. What I described above sounds more to me like a Raycast enhancement: any command hotkey should "reset" the Raycast hotkey workflow.

The feature request was more about: let's suppose I don't have hotkey for `Emoji Search`, then I would prefer the extension to pop to root after copying an emoji. Indeed, most of the time, I need one specific emoji from time to time so I don't need Raycast to stay on `Emoji Search` after I copy the emoji. I realize this is very subjective.
